modules/webhook/teamcity.go

```diff
@@ -0,0 +1,25 @@
+package webhook
+
+import (
+	"code.gitea.io/gitea/models"
+	"code.gitea.io/gitea/modules/log"
+	"encoding/json"
```

Comment:
Please split up imports with built in first, then an empty line before code.Gitea.io ones

Reply:
✔